src/librustdoc/clean/types.rs

```diff
@@ -2494,6 +2494,17 @@ impl Import {
     pub(crate) fn new_glob(source: ImportSource, should_be_displayed: bool) -> Self {
         Self { kind: ImportKind::Glob, source, should_be_displayed }
     }
+
+    pub(crate) fn imported_item_is_doc_hidden(&self, tcx: TyCtxt<'_>) -> bool {
+        match self.source.did {
+            Some(did) => tcx
+                .get_attrs(did, sym::doc)
```

Comment:
For some reason that I'm not sure of, using `.lists` here instead of open coding it causes panics. https://github.com/rust-lang/rust/pull/106538#issuecomment-1373996487

I think it's to do with the difference between `tcx.item_attrs` and `tcx.get_attrs`, as [this was the diff that fixed it.](https://github.com/rust-lang/rust/compare/98130d1b3af4e800bd6b1c5d5e09b525878c53f5..ff46d116c1fec1a1ceb988e0e31263c00a5d521b)